core/src/main/java/brooklyn/util/crypto/FluentKeySigner.java

```diff
@@ -42,7 +44,9 @@
  * X509v3CertificateBuilder drags in an add'l dependency (bcmail) and is harder to use. */
 @SuppressWarnings("deprecation")
 public class FluentKeySigner {
-    
+
+    static { Security.addProvider(new BouncyCastleProvider()); }
```

Comment:
Sounds sensible, I think. Is there anything beyond bouncy castle currently that we'd want to include in a `BrooklynInitialization`? If it's just bouncy castle, then maybe we live with code as-is for now, with an appropriate comment somewhere (e.g. `LocalManagementContext` or the `Main` class?) about this initialization.

I presume that would be at the level of the `ManagementContext` so wouldn't know about initializing web-apps etc?